seatunnel-e2e/seatunnel-connector-v2-e2e/connector-elasticsearch-e2e/src/test/resources/elasticsearch/elasticsearch_source_and_sink.conf

```diff
@@ -71,5 +71,7 @@ sink {
 
     index = "st_index2"
     index_type = "st"
+    "schema_save_mode"="CREATE_SCHEMA_WHEN_NOT_EXIST"
+    "data_save_mode"="APPEND_DATA"
```

Comment:
How can I test this option?

Reply:
You can turn off automatic index creation using the following


https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html#create-document-ids-automatically
```
PUT _cluster/settings
{
  "persistent": {
    "action.auto_create_index": "false" 
  }
}
```

Reply:
I see there is catalog testcase which can cover this test